app/src/forms/common/utils.js

```diff
@@ -361,12 +351,12 @@ const getSubmissionPeriodDates = (
   let graceDate = null;
 
   calculatedCloseDate.add(keepOpenForTerm, keepOpenForInterval);
-  if (allowLateTerm && allowLateInterval) graceDate = calculatedCloseDate.clone().add(allowLateTerm, allowLateInterval).format('YYYY-MM-DD HH:MM:SS');
```

Comment:
From here on out, removed a lot of the `.format` function because it would convert the moment object into a string, preventing us from using moment functions until we converted it back into a moment object.